src/app/pages/landing/landing.css

```
@@ -247,3 +261,39 @@
+  50% {
+    transform: scale(1.5);
+    opacity: 1;
+  }
+  100% {
+    transform: scale(1.5);
+    opacity: 0;
+  }
+}
+
+/* Optional: Style the child components for visibility */
+:host ::ng-deep app-global-map-2,
```

Biome (2.1.2)

[error] Unexpected unknown pseudo-elements: ng-deep

See MDN web docs for more details.
Use a known pseudo-elements instead, such as:


(lint/correctness/noUnknownPseudoElement)